src/style.css

```
@@ -30,27 +30,27 @@ body {
 }
 
 h1 {
-  @apply text-6xl lg:text-7xl;
+  @apply text-6xl ;
```

Just a tiny nit, but could you remove the whitespace between the class name and the semicolon?

```suggestion
  @apply text-6xl;
```